Doc/library/importlib.rst

```diff
@@ -775,6 +775,168 @@ ABC hierarchy::
+                     not have sub-resources (i.e. it cannot be a directory).
+    :type resource: ``Resource``
+    :param encoding: The encoding to open the resource in.  *encoding* has
+                     the same meaning as with :func:`open`.
+    :type encoding: str
+    :param errors: This parameter has the same meaning as with :func:`open`.
+    :type errors: str
+    :returns: an I/O stream open for reading.
+    :rtype: ``typing.TextIO``
+
+.. function:: read_binary(package, resource)
+
+    Read and return the contents of the *resource* within *package* as
+    ``bytes``.
+
+    :param package: A package name or module object.  See above for the API
+                    that such module objects must support.
+    :type package: ``Package``
+    :param resource: The name of the resource to read within *package*.
+                     *resource* may not contain path separators and it may
+                     not have sub-resources (i.e. it cannot be a directory).
+    :type resource: ``Resource``
+    :returns: the contents of the resource.
+    :rtype: ``bytes``
+
+.. function:: read_text(package, resource, encoding='utf-8', errors='strict')
+
+    Read and return the contents of *resource* within *package* as a ``str``.
+    By default, the contents are read as strict UTF-8.
+
+    :param package: A package name or module object.  See above for the API
+                    that such module objects must support.
+    :type package: ``Package``
+    :param resource: The name of the resource to read within *package*.
+                     *resource* may not contain path separators and it may
+                     not have sub-resources (i.e. it cannot be a directory).
+    :type resource: ``Resource``
+    :param encoding: The encoding to read the contents of the resource in.
+                     *encoding* has the same meaning as with :func:`open`.
+    :type encoding: str
+    :param errors: This parameter has the same meaning as with :func:`open`.
+    :type errors: str
+    :returns: the contents of the resource.
+    :rtype: ``str``
+
+.. function:: path(package, resource)
+
+    Return the path to the *resource* as an actual file system path.  This
+    function returns a context manager for use in a :keyword:`with` statement.
+    The context manager provides a :class:`pathlib.Path` object.
+
+    Exiting the context manager cleans up any temporary file created when the
+    resource needs to be extracted from e.g. a zip file.
+
+    :param package: A package name or module object.  See above for the API
+                    that such module objects must support.
+    :type package: ``Package``
+    :param resource: The name of the resource to read within *package*.
+                     *resource* may not contain path separators and it may
+                     not have sub-resources (i.e. it cannot be a directory).
+    :type resource: ``Resource``
+    :returns: A context manager for use in a :keyword`with` statement.
```

Comment:
souldn't this line read ``    :returns: A context manager for use in a :keyword:`with` statement.``?